checks/mongodb_instance

```diff
@@ -35,7 +35,12 @@ def inventory_mongodb_instance(info):
 
 def check_mongodb_instance(_no_item, _no_params, info):
     infos = ["%s: %s" % (line[0].title(), line[1]) for line in info]
-    yield 0, ", ".join(infos)
+    for line in info:
+        status, messg = line
+        if status == "error":
+            yield 2, messg
+        else:
+            yield 0, ", ".join(infos)
```

Comment:
Same as above, `yield 0,  mssg`, right?

Reply:
nope, beacuase it's a list of list

Reply:
You iterate over all lines in info, right? For each line, you check if there is an error. If there is an error, we get
`2, messg`
and I guess if not, we should get
`0, f"{status.title}: {messg}"`

Reply:
Looks good, please change it to accordingly and I will merge it subsequently.

Reply:
> PS: AFAIK String Interpolation is new in Python 3.6 but check_mk 1.6 use Python2

For this reason I forked only 1.6.0 branch.
So for the next times I guess if can I continue to work only on this branche (1.6.0) or not?

Reply:
But the question is: How can I controbute on werks or patchs for the 1.6.0?

Reply:
Thank Jörg for all your explainations and patience.
Just the last clarification. Do the approved changes end up in the 2.0.0 patches when working with the master?
Excuse my poor English